python/mathsys/dev/__init__.py

```
@@ -31,30 +34,53 @@ def timeWrapper(function):
+        unix_x86_64.cache_info()
+    ]
+
+#> MAIN -> CLEAR
+def clear() -> None:
+    validate.cache_clear()
+    latex.cache_clear()
+    web.cache_clear()
+    unix_x86_64.cache_clear()
+
 #> MAIN -> VALIDATE
+@lru_cache(maxsize = None)
```

Ruff (0.14.4)

Use `@functools.cache` instead of `@functools.lru_cache(maxsize=None)`

Rewrite with `@functools.cache

(UP033)